src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { css } from '@emotion/react';
+import { Link } from '@newrelic/gatsby-theme-newrelic';
+
+const Breadcrumbs = ({ segments, separator }) => {
+  return (
+    <div
+      css={css`
+        margin: 2em 0;
+        > :first-child {
+          margin-left: 0;
+        }
+        > * {
+          margin-left: 0.5em;
+        }
+      `}
+      aria-label="breadcrumb"
+    >
+      {segments.map((segment) => {
+        const elem = segment.url ? (
+          <Link to={segment.url}>{segment.name}</Link>
+        ) : (
+          segment.name
+        );
+        return (
+          <span
+            key={`breadcrumb-${segment.name}`}
+            css={css`
+              :after {
+                margin-left: 0.5em;
+                content: ' ${separator} ';
+              }
+              :last-child {
+                :after {
+                  content: '';
+                }
+              }
```

Comment:
There was [a great article](https://css-tricks.com/you-want-enabling-css-selectors-not-disabling-ones/) (and linked articles) published about this topic yesterday. The author proposes that we should strive for "enabling" selectors rather than creating a generic selector and then an additional one to "unset" a style.

Here's how the author would recommend this block. I also added some rules to prevent you from needing to add some whitespace _characters_ on either side of the separator.

```css
:not(:last-of-type) {
  margin-left: 0.5em;
  padding: 0 0.5em;
  display: inline-block;
  content: ${separator};
}
```

Reply:
oh nice. yeah, when i wrote that i thought it was a little weird / funky to have to undo it for the `:last-child`. added that with slight tweak